### PR TITLE
Always trigger alert

### PR DESCRIPTION
**Pull request by matrohon**

### CODE REVIEW

**Comment by safchain** [comment]
ok to test

**Comment by safchain** [comment]
add to whitelist

File: tests/alert_test.go [hunk 1]
```diff
@@ -320,3 +320,87 @@ func TestAlertWithTimer(t *testing.T) {
...
+			{"ip netns del alert-lo-down", true},
+		},
+
+		tearDownFunction: func(c *TestContext) error {
+			helper.WSClose(ws)
+			return c.client.Delete("alert", al.ID())
+		},
+
+		check: func(c *TestContext) error {
+			alert_number := 0
```
Comment (safchain) [inline tests/alert_test.go]: nit : alert_number should be alertNumber

**Comment by safchain** [comment]
Just a tiny comment, otherwise LGTM

**Comment by matrohon** [comment]
closing in favor of the PR from Orange-OpenSource repo : #297

**Pull request closed without merging**